.codecov.yml

```diff
@@ -1,14 +1,27 @@
-comment: off
+# See https://docs.codecov.com/docs/pull-request-comments
+# comment: false
 
+# See https://docs.codecov.com/docs/coverage-configuration
 coverage:
   range: 20..100
   round: down
   precision: 2
 
+# See https://docs.codecov.com/docs/commit-status
   status:
     project: off
-    patch: off
+    # patch = enable measures on lines adjusted in the PR only.
+    patch:
+      default:
+        # target = choose a minimum coverage ratio that the commit must meet to be considered a success.
+        # target: auto = compare against the coverage of the PR parent commit.
+        target: auto
+        # threshold = Allow the coverage to drop by <number>%, and posting a success status.
+        threshold: 0%
+        # Will set the status to success even if the CI fails
+        if_ci_failed: success
 
+# See https://docs.codecov.com/docs/ignoring-paths
 ignore:
   - "pkg/trace/testutil"  # code only used in tests
-  - "pkg/trace/log"       # logging utilities with no customer facing impact. Nothing to test.
\ No newline at end of file
+  - "pkg/trace/log"       # logging utilities with no customer facing impact. Nothing to test.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏

Reply:
Ahahahah 🫰